api/envoy/config/bootstrap/v3/bootstrap.proto

```diff
@@ -734,3 +738,14 @@ message CustomInlineHeader {
   // The type of the header that is expected to be set as the inline header.
   InlineHeaderType inline_header_type = 2 [(validate.rules).enum = {defined_only: true}];
 }
+
+message MemoryAllocatorManager {
+  // Configures tcmalloc to perform background release of free memory in amount of bytes per ``memory_release_interval`` interval.
+  // If equals to ``0``, no memory release will occur. Defaults to ``0``.
+  uint64 bytes_to_release = 1 [(validate.rules).uint64 = {gte: 1}];
```

Comment:
PGV here says that is >= 1 which contradicts above? Can you fix one or the other?